src/index.html

```diff
@@ -26,15 +18,6 @@
     <!-- add to homescreen for ios -->
     <meta name="apple-mobile-web-app-capable" content="yes" />
     <meta name="apple-mobile-web-app-status-bar-style" content="black" />
```

Comment:
I think we can remove these meta tags as well as they were needed for angular ionic.

Reply:
thanks for spotting this 👍
done (squashing and rebasing)